docs/tutorial_iteration.rst

```diff
@@ -101,5 +101,25 @@ the bounds, the loop will start on the first day of data and end on the last day
    mean_dB.plot(title='Absolute Daily Mean of ' + variable_str)
    plt.ylabel('Absolute Daily Mean ('+ units_str +')')
```

Comment:
At the top of this tutorial, `start` and `stop` need to be defined, all necessary modules imported, and the necessary data needs to be downloaded.  The load command should also probably be altered or text added above the example that explains why only one day is being loaded at a time.

Reply:
As is, I currently can't run these examples.  Providing the start and stop dates should fix that.  I tried 1-3/1/2009 and it didn't work.